maint/tuning/coll/ch4/generic.json

```diff
@@ -0,0 +1,230 @@
+{
+    "collective=bcast":
+    {
+        "comm_type=intra":
+        {
+            "comm_hierarchy=parent":
+            {
+               "comm_size<8":
+               {
+                   "avg_msg_size=any":
+                   {
+                       "composition=MPIDI_Bcast_intra_composition_alpha":{}
+                   }
+               },
+               "comm_size=pow2":
+               {
+                   "avg_msg_size<12288":
+                   {
+                       "composition=MPIDI_Bcast_intra_composition_alpha":{}
+                   },
+                   "avg_msg_size<524288":
+                   {
+                       "composition=MPIDI_Bcast_intra_composition_beta":{}
+                   },
+                   "avg_msg_size=any":
+                   {
+                       "composition=MPIDI_Bcast_intra_composition_gamma":{}
+                   }
+               },
+               "comm_size=any":
+               {
+                   "avg_msg_size<12288":
+                   {
+                       "composition=MPIDI_Bcast_intra_composition_alpha":{}
+                   },
+                   "avg_msg_size=any":
+                   {
+                       "composition=MPIDI_Bcast_intra_composition_gamma":{}
+                   }
+               }
+            },
+            "comm_hierarchy=node_roots":
```

Comment:
How about `"comm_hierarchy=any":` to avoid code duplication

Reply:
Good point.  I'll fix it.

Reply:
Fixed.